runtime/src/accounts_index.rs

```diff
@@ -212,9 +216,16 @@ impl AccountMapEntryMeta {
 }
 
 #[derive(Debug, Default)]
+/// one entry in the in-mem accounts index
+/// key is Pubkey
+/// this is the value
 pub struct AccountMapEntryInner<T> {
+    /// number of active slots that contain >= 1 instances of account data for this pubkey
     ref_count: AtomicU64,
+    /// list of slots that contain account data for this pubkey
+    /// Note that 'clean' removes outdated entries (ie. older roots) from this slot_list
```

Comment:
`purge_slot()` also removes non-rooted slots from this list

Reply:
updated